### PR TITLE
[TOOLS-2086] Add server6.1 metrics.

### PR DESCRIPTION
**Pull request by dwelch-spike**

### CODE REVIEW

File: watcher_jobs.go [hunk 1]
```diff
@@ -112,7 +112,7 @@ func (jw *JobsWatcher) refresh(o *Observer, infoKeys []string, rawMetrics map[st
 				continue
 			}
 
-			ch <- prometheus.MustNewConstMetric(pm.desc, pm.valueType, pv, rawMetrics[ikClusterName], rawMetrics[ikService], stats["ns"], stats["set"], stats["module"], stats["job-type"], stats["trid"])
+			ch <- prometheus.MustNewConstMetric(pm.desc, pm.valueType, pv, rawMetrics[ikClusterName], rawMetrics[ikService], stats["ns"], stats["set"], stats["module"], stats["job-type"], stats["trid"], stats["sindex-name"])
```
Comment (spkesan) [inline watcher_jobs.go:115]: From where are we getting sindex-name ?

Comment (spkesan) [reply watcher_jobs.go:115]: Well not exactly a breaking change. But the label 'sindex_name' could go empty for older server releases.

Comment (dwelch-spike) [reply watcher_jobs.go:115]: @spkesan I thought that an empty label would be fine. Do you suggest we only add some logic to only add the label for 6.1+ servers?

Comment (spkesan) [reply watcher_jobs.go:115]: Oh, you're right. I think we are good here.
https://prometheus.io/docs/concepts/data_model/
> _A label with an empty label value is considered equivalent to a label that does not exist._

**Review by spkesan — APPROVED** [review]
LGTM

**Pull request merged**